low-level/files/opt/seagate/sspl/bin/constants.sh

```diff
@@ -26,7 +26,7 @@ SSPL_STORE_TYPE="consul"
 CONSUL_HOST="127.0.0.1"
 CONSUL_PORT="8500"
 ENVIRONMENT="PROD"
-CONSUL_PATH="/opt/seagate/$PRODUCT_FAMILY/hare/bin"
-CONSUL_FALLBACK_PATH="/opt/seagate/$PRODUCT_FAMILY/sspl/bin"
+CONSUL_PATH="/usr/bin"
```

Comment:
Keep last slash also in path so we dont need to add it explicitly.
> CONSUL_PATH="/usr/bin/"

And in following case, keep blank so consul moving to any path & that path present in env var $PATH would work
CONSUL_FALLBACK_PATH=""

Reply:
CONSUL_PATH="/usr/bin/"
CONSUL_FALLBACK_PATH=""

The usage of this variable in our sh scripts is like, 
`$CONSUL_PATH/consul kv put sspl/config/RABBITMQCLUSTER/cluster_nodes $pout`
then the usage would be,
`${CONSUL_PATH}consul kv put sspl/config/RABBITMQCLUSTER/cluster_nodes $pout`

Is this fine?

Reply:
We don't need CONSUL_PATH at all? just `consul` will work fine as it is in /usr/bin.

Reply:
Agree if consul will be available in /usr/bin ever. But how stable consul location will be in future.
Not sure whether if this consul binary can be moved back to Hare or any other location later.
@thavanathan

Reply:
@mariyappanp yes for 
>${CONSUL_PATH}consul kv put sspl/config/RABBITMQCLUSTER/cluster_nodes $pout

Specially it would help with 
> And in following case, keep blank so consul moving to any path & that path present in env var $PATH would work
CONSUL_FALLBACK_PATH=""

@thavanathan ideally yes we dont need, as standard PATH does have /usr/bin included, but too much dependency on $PATH is also not good from security perspective, this helps if PATH is prepended with some other path to use malware kind of consul,etc. Also we are reusing some old code to keep some additional checks & balances in case onsul path not there. and fallback path applies to what you are saying, using $PATH locations

Reply:
As discussed, adhere to minimal change I removed CONSUL_FALLBACK_PATH and kept only CONSUL_PATH='usr/bin/'.